project.clj

```diff
@@ -1,11 +1,12 @@
-(defproject org.cyverse/clojure-commons "3.0.10-SNAPSHOT"
+(defproject org.cyverse/clojure-commons "3.0.11-SNAPSHOT"
```

Comment:
`org.cyverse/clojure-commons` version `3.0.10` already exists on Clojars, so I bumped this up to `3.0.11-SNAPSHOT`